roles/secretgen_controller/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2022 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Deploy secretgen-controller
+  kubernetes.core.k8s:
+    state: present
+    template: release.yml.tpl
```

Comment:
Is there any reason this is templated?  You could use `src` if it's not templated and I don't see any templates in there.

Also, is it possible to use `vendir` to sync that file?

Reply:
for image ref, i used template